I have started looking at this PR and noticed that the Unit tests failed because of the following error:

```
client/reader/site-subscriptions-manager/site-subscriptions-manager.tsx(5,2): error TS2724: '"@wordpress/components"' has no exported member named '__experimentalSpacer'. Did you mean '__experimentalHStack'?
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -493,6 +493,7 @@ module.exports = {
 					'__experimentalDivider',
 					'__experimentalHStack',
 					'__experimentalVStack',
+					'__experimentalSpacer',
 					'__experimentalItem',
 					'__experimentalItemGroup',
 					'__experimentalNavigationBackButton',

--- a/client/reader/site-subscriptions-manager/site-subscriptions-manager.tsx
+++ b/client/reader/site-subscriptions-manager/site-subscriptions-manager.tsx
@@ -1,7 +1,6 @@
 import {
 	__experimentalHStack as HStack,
 	__experimentalVStack as VStack,
-	// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
 	__experimentalSpacer as Spacer,
 } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';

--- a/client/wp-components-types.d.ts
+++ b/client/wp-components-types.d.ts
@@ -8,7 +8,6 @@ declare module '@wordpress/components' {
 	export const __experimentalDivider: React.ComponentType< Props >;
 	export const __experimentalHStack: React.ComponentType< Props >;
 	export const __experimentalVStack: React.ComponentType< Props >;
-
 	export const __experimentalSpacer: React.ComponentType< Props >;
 	export const __experimentalItem: React.ComponentType< Props >;
 	export const __experimentalItemGroup: React.ComponentType< Props >;

--- a/client/wp-components-types.d.ts
+++ b/client/wp-components-types.d.ts
@@ -8,6 +8,8 @@ declare module '@wordpress/components' {
 	export const __experimentalDivider: React.ComponentType< Props >;
 	export const __experimentalHStack: React.ComponentType< Props >;
 	export const __experimentalVStack: React.ComponentType< Props >;
+
+	export const __experimentalSpacer: React.ComponentType< Props >;
 	export const __experimentalItem: React.ComponentType< Props >;
 	export const __experimentalItemGroup: React.ComponentType< Props >;
 	export const __experimentalNavigatorBackButton: React.ComponentType< Props >;

